simbad/util/pyrvapi_results.py

```diff
@@ -105,13 +105,14 @@ class SimbadOutput(object):
                         "Number_of_rotation_searches_producing_peak": "Number of rotations searches which produce "
                         "each peak [out of 5]"}
 
-    def __init__(self, rvapi_document, webserver_uri, display_gui, logfile, work_dir, ccp4i2_xml=None):
+    def __init__(self, rvapi_document, webserver_uri, display_gui, logfile, work_dir, ccp4i2_xml=None, tab_prefix=None):
```

Comment:
Default ``tab_prefix=""``, basically an empty string. Then you don't need to check for ``is None`` but instead can just blindly prefix each tab name.